### ISSUE TITLE
VoiceOver does not announce confirmation message after creating/deleting or revoking an employee account

### ISSUE THREAD

**Issue by vanpham-sw** [shopware/shopware#7822]
### PHP Version

8.2.25

### Shopware Version

6.7

### Affected area / extension

Extension:Commercial

### Actual behaviour

VoiceOver does not read the confirmation message, leaving the user unaware of the successful employee **creation/deletion/revocation** and the next steps.

### Expected behaviour

Upon successful creation of a new employee account, VoiceOver should announce the confirmation message: "A new employee account has been created. This account has to be activated by the employee via the link in the invitation email."

### How to reproduce

1. Enable VoiceOver on the device.
2. Navigate to the "Create New Employee" page in the application.
3. Fill in the required fields to create a new employee.
4. Click the "Save" button to submit the form.
